### PR TITLE
Expiration days configure

### PR DESCRIPTION
**Pull request by benjwadams**

### CODE REVIEW

File: catalog_harvesting/cli.py [hunk 1]
```diff
@@ -50,7 +50,11 @@ def main():
 
     if args.force_clean and args.dest:
         get_logger().info("Removing stale datasets")
-        force_clean(args.dest)
+        max_days = os.getenv('STALE_EXPIRATION_DAYS', None)
+        if max_days is not None:
+            force_clean(args.dest, max_days)
+        else:
+            force_clean(args.dest)
```
Comment (lukecampbell) [inline catalog_harvesting/cli.py]: You could shorten this to 
```
force_celan(args.dest, max_days or 3)
```

Also, STALE_EXPIRATION_DAYS is a `str` so if it is actually set this will break. You need to parse out anything that's not a string.

File: catalog_harvesting/cli.py [hunk 2]
```diff
@@ -50,7 +50,11 @@ def main():
 
     if args.force_clean and args.dest:
         get_logger().info("Removing stale datasets")
-        force_clean(args.dest)
+        max_days = os.getenv('STALE_EXPIRATION_DAYS', None)
```
Comment (lukecampbell) [inline catalog_harvesting/cli.py]: This should be:

```python
max_days = os.getenv('STALE_EXPIRATION_DAYS', None)
try:
    max_days = int(max_days)
except ValueError: # I think it's a value error
    pass
```

**Review by lukecampbell — CHANGES_REQUESTED** [review]

**Comment by benjwadams** [comment]
@lukecampbell, can you re-review?

**Review by lukecampbell — APPROVED** [review]
